examples/stimuli/tracker_dealer.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+======================
+Tracker Dealer Example
+======================
+
+This file shows how to interleave multiple Tracker objects using
+:class:`expyfun.stimuli.TrackerDealer`.
+
+In this case, a modeled human subject generates two curves (one for each trial
+type: 1 & 2)
+
+@author: maddycapp27
+"""
+
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerDealer
+from expyfun.analyze import sigmoid
+import matplotlib.pyplot as plt
+
+# define parameters of modeled subject (using sigmoid probability)

```

Comment:
FYI if you want you can make pretty sections in the output like this:

https://github.com/mne-tools/mne-python/blob/master/tutorials/plot_artifacts_correction_filtering.py#L49

that render like this:

https://mne-tools.github.io/dev/auto_tutorials/plot_artifacts_correction_filtering.html#removing-power-line-noise-with-notch-filtering

might be overkill in this example but it's something to consider when writing pedagogical examples